frontend/src/metabase/core/components/BookmarkToggle/BookmarkToggle.tsx

```diff
@@ -44,10 +38,9 @@ const BookmarkToggle = forwardRef(function BookmarkToggle(
 
   return (
     <Tooltip tooltip={isBookmarked ? t`Remove from bookmarks` : t`Bookmark`}>
-      <IconWrapper
-        {...props}
+      <BookmarkButton
         ref={ref}
-        hover={{ color: isBookmarked ? color("brand") : color("text-dark") }}
+        hoverColor={isBookmarked ? color("brand") : color("text-dark")}
```

Comment:
Can we pass `isBookmarked` to the styled component instead? We try to use `color` in styled components only

Reply:
Fixed.